certified-connectors/Secib/apiProperties.json

```diff
@@ -0,0 +1,50 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "clientId": "{Application (client) ID of the connector in Azure App Registrations}",
+          "scopes": [
+            "{API Permissions of the connector in Azure App Registrations}"
```

Comment:
@thomasBeckSecib Please make sure you are providing valid scopes while submitting the connector into ISV Studio for certification.

Reply:
Every scopes are now provided in connector artifacts but without the clientId that composed some of them.

Reply:
Scopes are valid now.